core/shared/src/main/scala/scalaz/zio/duration/DurationConversions.scala

```diff
@@ -0,0 +1,37 @@
+package scalaz.zio.duration
+
+import java.util.concurrent.TimeUnit
+import java.util.concurrent.TimeUnit._
+
+trait DurationConversions extends Any {
```

Comment:
Looks good. We have all the same abbreviations as Scala?

Reply:
Yes we do  https://github.com/scala/scala/blob/v2.12.7/src/library/scala/concurrent/duration/DurationConversions.scala